app/views/_mobile-service-instance-row.html

```diff
@@ -39,7 +39,7 @@ <h3>
     <div class="list-pf-container">
       <div class="expanded-section no-margin">
         <div class="row" ng-if="(row.bindings | size) && row.bindingInProgress == false">
-          <div class="col-md-6">
+          <div ng-class="{'col-md-6' : row.serviceType === 'ups', 'col-md-12' : row.serviceType !== 'ups'}">
```

Comment:
Can this be displayed based on `row.extendedAnnotations`? It will probably mean adding a new container.

```.html
<div ng-class="{'col-md-6' : row.extendedAnnotations, 'col-md-12' : !row.extendedAnnotations}">
///
</div>
<div class="col-md-6" ng-if="row.extendedAnnotations">
  <div class="col-md-12" ng-if="row.serviceType === 'ups'">
    <h4 class="component-label section-label">Variants</h4>
    <dl class="dl-horizontal left">
      <dt ng-repeat-start="variant in row.extendedAnnotations.variants">{{variant.typeLabel}}</dt>
      <dd ng-repeat-end>
        <a href="{{variant.url}}">{{ variant.id }}</a>
     </dd>
    </dl>
  </div>
</div>
```